bosh/releases/pre_render_scripts/routing-api/routing-api/bpm/patch_bpm.sh

```diff
@@ -1,5 +1,18 @@
 #!/usr/bin/env bash
 
+# This patch changes the BPM command to not use spec.ip because the
+# quarks-operator implements process management very differently from
+# BOSH: BPM is rendered from a completely different container (and
+# therefore has no chance of having a valid spec.ip.
+#
+# We have no idea how we can implement the change in an upstream-
+# compatible way: we hack around things by changing the POD_IP
+# environment variable via an ops file, which wouldn't make any sense
```

Comment:
It would be good to link to the ops file, or at least give its file name.

Reply:
Will do.

Reply:
Done with commit 7d56fbc4d2f056f8691d2eb4ab2418d6506721d8